include/libc/assert.h

```diff
@@ -0,0 +1,45 @@
+#ifndef ASSERT_H
+#define ASSERT_H
+
+#if !defined(__GNUC__) && !defined(__attribute__)
+#define __attribute__(x)
+#endif
```

Comment:
Wouldn't be better to include `attributes.h` instead of duplicating this define all over the place?

Reply:
I'd rather not include non-libc headers in libc headers. Same reason for why I use `__attribute__((noreturn))` above